README.md

```diff
@@ -1,95 +1,99 @@
-MindRacer user manual:
-http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf
+The designs listed below (with a high level overview of the main differences).
 
-Key features of v1.2:
+Version | Description | Design | Hardware | Description
+--- | ---
+[FMUv1](FMUv1/README.md) & [IOv1](IOv1) | (Discontinued) Original Flight Management Unit and Separate I/O board.
+[FMUv2](FMUv2/README.md) | Single board with STM32427VI processor.
+[FMUv3](FMUv3_REV_D/README.md)| Identical to FMUv2, but usable flash doubled to 2MB.
+[FMUv4](FMUv4/README.md) | Increased RAM. Faster CPU. More serial ports. No IO processor
+FMUv4-PRO | Slightly increased RAM. More serial ports. IO processor.
+[FMUv5](FMUv5/README.md) | New processor (F7). Much faster. More RAM. More CAN busses. Much more configurable.<br>> **Note** Minimum specification provided (pinout info, but no schematics).
 
-1. 35 x 35mm (30.5 mm mounting holes) ultra mini size, weighs about 6g
-2. STM32F427 as processor
-3. 6 main PWM outputs
-4. onboard MPU6500 & HMC5883L
-5. dual 2x7 stackable header pins for I/O port replication/expansion
-6. optional IMU isolation mounting support
-7. Support PPM/SBUS/SPEKTRUM DSM/Satellite RC input
-8. Support FrSky/Wifi telemetry
-9.  onboard RTC battery and TF card slot
-10. DroneCode standard connectors
 
-Note: MindPX v2.8 & MindRacer v1.2 share same firmware.
 
-[Sapog ESC](https://github.com/PX4/sapog) Hardware
-==================================================
+### Sapog ESC
 
-The directory `sapog_reference_hardware` contains the reference hardware design for the
-Sapog ESC firmware. Refer to its README.md for more info.
+- [Sapog ESC](https://github.com/PX4/sapog) Hardware - The directory [sapog_reference_hardware](sapog_reference_hardware/README.md) contains the reference hardware design for the Sapog ESC firmware.
 
-Creative Commons License Disclaimer
-===================================
+### Other
 
-UNLESS OTHERWISE MUTUALLY AGREED TO BY THE PARTIES IN WRITING, LICENSOR OFFERS THE WORK AS-IS AND MAKES NO REPRESENTATIONS OR WARRANTIES OF ANY KIND CONCERNING THE WORK, EXPRESS, IMPLIED, STATUTORY OR OTHERWISE, INCLUDING, WITHOUT LIMITATION, WARRANTIES OF TITLE, MERCHANTIBILITY, FITNESS FOR A PARTICULAR PURPOSE, NONINFRINGEMENT, OR THE ABSENCE OF LATENT OR OTHER DEFECTS, ACCURACY, OR THE PRESENCE OF ABSENCE OF ERRORS, WHETHER OR NOT DISCOVERABLE. SOME JURISDICTIONS DO NOT ALLOW THE EXCLUSION OF IMPLIED WARRANTIES, SO SUCH EXCLUSION MAY NOT APPLY TO YOU.
-EXCEPT TO THE EXTENT REQUIRED BY APPLICABLE LAW, IN NO EVENT WILL LICENSOR BE LIABLE TO YOU ON ANY LEGAL THEORY FOR ANY SPECIAL, INCIDENTAL, CONSEQUENTIAL, PUNITIVE OR EXEMPLARY DAMAGES ARISING OUT OF THIS LICENSE OR THE USE OF THE WORK, EVEN IF LICENSOR HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
+- [AIRSPEEDv1](AIRSPEEDv1) - 
+- [FLOWv1](FLOWv1)
+- [IMUv3_REV_C](IMUv3_REV_C)
+- [PSMv3_REV_C](PSMv3_REV_C)
 
-http://creativecommons.org/licenses/by-sa/3.0/
+
+## Derived Autopilot Products
```

Comment:
@bkueng This is supposed to be section for products that are derived from the FMU designs and need to provide their own specs etc. Are there any others missing? 

Re the title - I think derived is legally good, but perhaps a little confusing because a vehicle that is compatible must in reality have been derived in some way.  Are you OK with this/can you think of anything better.

Reply:
I'm not aware of others.

I agree it could be confusing, but it's correct, and I don't have anything better.